src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -514,7 +514,7 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             cppkafka::Configuration configs = {{"metadata.broker.list", "127.0.0.1:9092"},
                                                {"group.id",             "knnect"}};
             KafkaConnector kstream(configs);
-            int numberOfPartitions = 4;
+            int numberOfPartitions = 1;
```

Comment:
why do we hard code number of partitions here?

Reply:
Fixed with https://github.com/miyurud/jasminegraph/pull/131/commits/7b7c883e03d54d5b3b8d565c3934d67305a111b1